cmd/crio/main.go

```diff
@@ -229,9 +232,9 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:  "log-level",
-			Usage: "log messages above specified level: debug, info (default), warn, error, fatal or panic",
+			Value: "error",
```

Comment:
are we sure about this default? what was before this change overall?

Reply:
it was info. We want the logs to be as minimal as possible by default so error seems like the best choice. When we change this in crio.conf the logs get more verbose for both cri-o and conmon.

Reply:
awesome, we're good then (error's super fine!)